packages/toolpad-app/src/toolpad/propertyControls/GridColumns.tsx

```diff
@@ -188,16 +189,20 @@ function GridColumnsPropEditor({
           <React.Fragment>
             <DialogTitle>Edit columns</DialogTitle>
             <DialogContent>
-              <IconButton
-                aria-label="Recreate columns"
-                onClick={handleRecreateColumns}
-                disabled={!hasColumnSuggestions}
-              >
-                <RefreshIcon />
-              </IconButton>
-              <IconButton aria-label="Add column" onClick={handleMenuClick} disabled={disabled}>
-                <AddIcon />
-              </IconButton>
+              <Tooltip title="Recreate columns">
+                <IconButton
+                  aria-label="Recreate columns"
```

Comment:
Tooltip [should add the label](https://mui.com/material-ui/react-tooltip/#accessibility)

Reply:
cool i didn't know, i've removed the unnecessary props + added a span so that the tooltip shows when the "recreate columns" button is disabled (as recommended by the MUI console warnings)